docs/source/widgets_documentation/ecephys.rst

```diff
@@ -0,0 +1,159 @@
+Use Case: Electrophysiology data
```

Comment:
I'm confused by these headings. You have 

> Use Case: Electrophysiology data

followed by the sub-heading

> Using optical physiology data:

Also, I don't want so much code just for the user to generate example data. I'd rather stream data from dandi [like this
](https://pynwb.readthedocs.io/en/stable/tutorials/advanced_io/streaming.html#sphx-glr-tutorials-advanced-io-streaming-py)